### PR TITLE
LPD-39121 Move Private Pages FF from System level to Intance Level

### PR DESCRIPTION
**Pull request by lfbesada**
Waiting for test fixes and bug present in Default Company.

https://liferay.atlassian.net/browse/LPD-39121

### CODE REVIEW

**Comment by liferay-page-management** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:test:page-management

**Comment by lfbesada** [comment]
ci:test:page-management-playwright

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -7521,6 +7521,8 @@ feature.flag.LPD-35678.description=Allow users on B2B and B2X channels to use gu
 feature.flag.LPD-35678.title=B2B Guest Checkout
 feature.flag.LPD-36010.description=Remove filtering by last login date for the the headless admin user REST API because indexing "lastLoginDate" for users negatively impacts search performance.
 feature.flag.LPD-36010.title=Remove Filter by Last Login Date for Headless Admin User
+feature.flag.LPD-38869.description=The use of Private Pages is deprecated. Please leverage the Content Pages to implement the experience.
```
Comment (ealonso) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:7524]: @lfbesada Private Pages and Content Pages should be in lower case

File: modules/test/playwright/tests/layout-set-prototype-web/siteTemplatesWithWebContent.spec.ts [hunk 2]
```diff
@@ -283,200 +283,200 @@ test('Can switch template with web content on widget page.', async ({
...
+		await layoutSetPrototypePage.checkIfWebContentAddedToHome(
+			siteName,
+			webContentText1
+		);
 
-	await applicationsMenuPage.goToServerAdministration();
+		await applicationsMenuPage.goToServerAdministration();
 
-	const script = `
+		const script = `
```
Comment (ealonso) [inline modules/test/playwright/tests/layout-set-prototype-web/siteTemplatesWithWebContent.spec.ts:457]: @lfbesada I think here you need to add a tab for script content, same for the other usage

**Comment by lfbesada** [comment]
I only solved conflicts, add baseline, and rebased

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
see: https://github.com/liferay-page-management/liferay-portal/pull/16530 https://github.com/liferay-page-management/liferay-portal/pull/16531

**Pull request closed without merging**